docs/user/exercises/programming.rst

```diff
@@ -121,6 +121,14 @@ for other programming languages.
 
 .. include:: test-case-best-practices.inc
 
+Best practices for creating AI-proof exercises
+----------------------------------------------
+
+The following section presents guidelines for crafting AI-resistant exercise instructions.
+While not every recommendation will suit every exercise, focus on those most pertinent to the specific problem at hand.
+This is illustrated with diverse exercise examples.
```

Comment:
Please consider rephrasing this sentence so that it is not written in a passive voice.